### PR TITLE
Fixed flake8 errors for the newest version of flake8

### PR DESCRIPTION
**Pull request by ryan-blakley**
* Added W605, and W504 to the ignores since it would require a lot of
  trivial changes to a lot of files.
* Fixed any flake8 errors thrown, most were indention, missing blank
  lines, and a few print formatting related issues.
* Re-ordered imports to be in alphabetical order on any files that I had
  to fix a flake8 error on.
* Bumped the flake8 version in the setup.py file.

Signed-off-by: Ryan Blakley <rblakley@redhat.com>

### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [X] Is this PR an enhancement?

### Complete Description of Additions/Changes:
Fixed any flake8 errors thrown by the latest version of flake8 3.9.1. Added W605, and W504 to ignores since it would require a lot of trivial changes to a lot of source files. Most errors were missing blank lines between functions, incorrect indention's, a few print formatting issues, and replacing is with == in a few tests. Also on any files I edited I re-ordered imports to be in alphabetical order.

### CODE REVIEW

File: insights/parsers/tests/test_parsers_module.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import pytest
+
 from collections import OrderedDict
-from insights.parsers import split_kv_pairs, unsplit_lines, parse_fixed_table
-from insights.parsers import calc_offset, optlist_to_dict, keyword_search
-from insights.parsers import parse_delimited_table, ParseException, SkipException
+from insights.parsers import calc_offset, keyword_search, optlist_to_dict, parse_delimited_table, parse_fixed_table,\
```
Comment (bfahr) [inline insights/parsers/tests/test_parsers_module.py:4]: I think `()` would be better here than `\`, it allows cleaner formatting of the imports.

**Review by bfahr — CHANGES_REQUESTED** [review]
Is your editor making some of these changes automatically such as reordering imports, or did flake8 flag those as errors?

**Comment by bfahr** [comment]
@ryan-blakley you'll also need to modify setup.py for the new version of `flake8` for the version of Python similar to the `[testing]` target.

File: insights/parsers/tests/test_parsers_module.py (hunk 1 above)
Comment (ryan-blakley) [reply insights/parsers/tests/test_parsers_module.py]: Ah good point, I updated it to that.

**Review by vdymna — APPROVED** [review]
This looks good to me.

**Pull request merged**